doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md

```diff
@@ -139,6 +140,13 @@ It is expected UTF-8 character set is maintained by all the Flume elements in th
 
 [Top](#top)
 
+### <a name="section2.3"></a>Quote escaping for `TextUnrestricted` attributes
+Cygnus escapes from `'` to `''` in attributes of type `TextUnrestricted` with the aim of avoiding injection attacks.
+
+Note that other attributes (i.e. with type different to `TextUnrestricted` don't need such escaping as single quote (`'`) is a [forbidden character in Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html) so that value will never arrives to Cygnus in notifications.
```

Comment:
Small typo:

```suggestion
Note that other attributes (i.e. with type different to `TextUnrestricted`) don't need such escaping as single quote (`'`) is a [forbidden character in Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html) so that value will never arrives to Cygnus in notifications.
```